lib/TInitialState.php

```diff
@@ -177,5 +177,10 @@ protected function publishInitialStateForGuest(): void {
 			'play_sounds',
 			false
 		);
+
+		$this->initialState->provideInitialState(
+			'ask_guest_username',
```

Comment:
please also publish this for logged in users with any value, just to make sure it doesn't yield an error in the console at some time.

Reply:
For logged in users SetGuestUsername.vue can not be used. Do you think, it is necessary?

Reply:
well the main issue is when at some point the component is mounted it will log an error already.

Reply:
I moved code to `publishInitialStateShared()`